packages/affiliates/libs/contracts.js

```diff
@@ -25,14 +31,51 @@ function DecodeTransaction(abi, meta = {}) {
   };
 }
 
-function decodeAttribution(
-  data,
-  delimiter = "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff000000000000000000000000"
-) {
-  assert(data, "requires data to decode");
-  return data.split(delimiter)[1];
+function GetInputLength(abi) {
```

Comment:
nit: should this be capital `G`?

Reply:
Id like to keep it capital since its more like a class which needs to be instanced with `abi` before it can be used.